src/Servers/Kestrel/Core/src/Internal/Http/HttpProtocol.cs

```diff
@@ -1171,7 +1173,7 @@ private HttpResponseHeaders CreateResponseHeaders(bool appCompleted)
                 {
                     if ((option.Protocols & HttpProtocols.Http3) == HttpProtocols.Http3)
                     {
-                        responseHeaders.HeaderAltSvc = $"h3-25=\":{option.IPEndPoint.Port}\"; ma=84600";
+                        responseHeaders.HeaderAltSvc = $"h3-25=\":{option.IPEndPoint!.Port}\"; ma=84600";
```

Comment:
Do we know that option.IPEndPoint isn't null here @jkotalik? Now that IMultiplexedConnectionListenerFactory is experimentally public, I figure it's theoretically possible. I might just add a null check on option.IPEndPoint in the above if condition.

Reply:
I'm going to leave this as it. This is experimental, and at the moment I think an exception to say that something is wrong is preferable to silently doing the wrong thing.